docs/3.0.0-beta.x/installation/docker.md

```diff
@@ -0,0 +1,102 @@
+# Installing using Docker
+
+If you're already familiar with Docker, you are probably looking for our official Docker images over [Docker Hub](https://hub.docker.com/r/strapi/strapi).
+
+[[toc]]
+
+## Step 1: Create a `docker-compose.yaml` file
```

Comment:
You are not explaining that you need a project that exists here.

Reply:
Hum nop, because it's not needed. 
If no project exist, the image will create one.

Reply:
It's still missing an explanation to create an empty folder so that the project will be created in it. You are mounting a volume with no explaining.

Reply:
Done